Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift

```diff
@@ -9,7 +9,33 @@ import UIKit
 /// A view representing a switch item.
 /// :nodoc:
 public final class FormToggleItemView: FormValueItemView<Bool, FormToggleItemStyle, FormToggleItem> {
-    
+
+    // MARK: - UI elements
+
+    private lazy var stackView: UIStackView = {
```

Comment:
I thought this was done in another PR?

Reply:
Yes, the fix is on dev already.

Reply:
I would advise updating to latest `develop` then 😆